workflows/rnaseq/downstream/rnaseq.Rmd

```diff
@@ -478,10 +498,12 @@ for (name in names(res.list)){
   dds.i <- res.list[[name]][['dds']]
   res.i <- res.list[[name]][['res']]
   label <- res.list[[name]][['label']]
+  alpha.i <- res.list[[name]][['alpha']]
+  lfc.thresh.i <- res.list[[name]][['lfc.thresh']]
 
   mdcat('## ', label, ' {.tabset}')
   mdcat('### Summary of results')
-  print(knitr::kable(my.summary(res.i, dds.i)))
+  print(knitr::kable(my.summary(res.i, dds.i, alpha.i, lfc.thresh=lfc.thresh.i)))
```

Comment:
following up on the above comments, `my.summary` can figure out the alpha and lfc thresh on its own which would simplify this